src/component/footer/footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./footer.module.scss";
+
+const Footer = ({
+  handleClickBack,
+  handleClick,
+  previousPageLink,
+  nextPageLink,
+}) => {
```

Comment:
If there is no sub-function inside a function, please code like this:
`
const Footer = ({
  handleClickBack,
  handleClick,
  previousPageLink,
  nextPageLink,
}) => (
    <footer className={styles.footer}>
`

Reply:
Which plugin do you use? I tried "prettier", but it didn't have settings to format the sub-function.

Reply:
I did it manually.

Reply:
Noted, I will do it manually. Thank you!